corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -154,7 +154,6 @@ hqDefine("cloudcare/js/formplayer/menus/api", [
                             if (parsedMenus.metaData) {
                                 attemptRestore = parsedMenus.metaData.attemptRestore;
                             }
```

Comment:
Could you also take out these lines?

Reply:
Will be keeping that code in now but just gate it behind the FF.